docs/root/configuration/http/http_filters/local_rate_limit_filter.rst

```diff
@@ -0,0 +1,51 @@
+.. _config_http_filters_local_rate_limit:
```

Comment:
Can you add some discussion of this feature to https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/local_rate_limiting and then link to here?

Reply:
Yup

Reply:
something like 9c010be perhaps?

Reply:
Yup that's great. Would it be possible to add a small sample config for the filter at both the global level as well as the per-route level?

Reply:
done